### PR TITLE
Refactor the errorMiddleware

### PR DESCRIPTION
**Pull request by vademo**
- put errorMiddeware in a seperate file
- print out unhandeld rejections

### CODE REVIEW

File: server/middlewares/error.middleware.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * error.middleware.js
+ * server/middlewares
+ *
+ */
+const { ErrorSerializer } = require('jsonade');
+const Errors = require('../errors');
+const { isArray } = require('lodash');
+
+// eslint-disable-next-line no-unused-vars
+function errorHandeler(err, req, res, next) {
```
Comment (hoetmaaiers) [inline server/middlewares/error.middleware.js]: I believe it is `Handler`, this sounds like Frenglish...

File: app.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const { isArray } = require('lodash');
 const loggerMiddleware = require('./server/middlewares/logger.middleware');
 const { getSwaggerDocument } = require('./server/helpers/swagger.helper');
 const Errors = require('./server/errors');
-const { ErrorSerializer } = require('jsonade');
+const Errorhandler = require('./server/middlewares/error.middleware');
```
Comment (hoetmaaiers) [inline app.js]: It is no class so camelCase in stead of PascalCase?

File: server/middlewares/error.middleware.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/**
+ * error.middleware.js
+ * server/middlewares
+ *
+ */
+const { ErrorSerializer } = require('jsonade');
+const Errors = require('../errors');
```
Comment (hoetmaaiers) [inline server/middlewares/error.middleware.js]: Shouldn't this be camelCased?

File: server/middlewares/error.middleware.js (hunk 1 above)
Comment (vademo) [reply server/middlewares/error.middleware.js]: j'ai enlevé mon frenlish ;)

File: app.js (hunk 2 above)
Comment (vademo) [reply app.js]: fixed.

File: server/middlewares/error.middleware.js (hunk 3 above)
Comment (vademo) [reply server/middlewares/error.middleware.js]: My bad I thought tis would be a constructor

**Review by hoetmaaiers — APPROVED** [review]

**Pull request merged**